extensions/blocks/subscriptions/save.js

```diff
@@ -1,17 +1,129 @@
 /**
  * External dependencies
  */
+import { select } from '@wordpress/data';
 import { RawHTML } from '@wordpress/element';
+import {
+	getColorClassName,
+	getColorObjectByAttributeValues,
+	__experimentalGetGradientClass as getGradientClass,
```

Comment:
We won't be able to use this here, as it breaks the editor in WordPress 5.3.2:

```
Uncaught (in promise) TypeError: Object(...) is not a function at save (editor.js?ver=1586966413:formatted:16337)
```

Reply:
Fixed with a gradient check.